app/services/audit/sdr_warc_lister.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Audit
+  # Enumberable that returns WARC filenames for a collection from SDR.
+  class SdrWarcLister
+    include Enumerable
+    def initialize(collection_druid:)
+      @collection_druid = collection_druid
+    end
+
+    def each(&block)
+      if block_given?
+        member_druids.each do |member_druid|
+          files_for(member_druid).each { |file| block.call(file) }
+        end
+      else
+        to_enum(:each)
+      end
+    end
+
+    private
+
+    attr_reader :collection_druid
+
+    def member_druids
+      Dor::Services::Client.object(collection_druid).members.map(&:externalIdentifier)
+    end
+
+    def files_for(member_druid)
+      cocina_obj = Dor::Services::Client.object(member_druid).find
+      return [] if cocina_obj.type == Cocina::Models::ObjectType.webarchive_seed
```

Comment:
I was thinking about that and the thing I got hung up on was whether we have collections where we're using AIT now, but previously used Heritrix, so there's a mix of files and some are going to show as missing in the audit results since they're not in AIT. But maybe that is not realistic or too much of an edge case.

Reply:
Since the question is "what does AIT have that Stanford doesn't have?" it shouldn't matter if a local Heritrix crawl is only in SDR, should it? I don't think we can use SDR to determine if AIT itself is missing anything, only if Stanford is missing something AIT definitely has.

Reply:
In terms of distinguishing where the WARCs are in SDR, I wouldn't consider anything other than filename to be reliable. If we can identify and then index WARC-containing druids as webarchive-binary, then we could limit the audit to those items.

Reply:
Yes, I'm sorry, I'm making it more complicated than required. If it's only in SDR, doesn't matter for this task and won't show up in the results.